src/components/Label/Label.tsx

```diff
@@ -0,0 +1,39 @@
+"use client";
+
+import React from 'react';
+import {clsx} from 'clsx';
+
+interface LabelProps {
+    text: string;
```

Comment:
현재 `text `필드가 `string`으로 되어있는데, `children`으로 받는것은 어떤지 제안해봅니다.
나중에 Label 안에 `<span>`이나 `<strong>` 같은 포맷팅 요소 넣고 싶을 때 `text`는 문자열만 받으므로 한계가 있을 수 있습니다. @Bangdayeon 의견 부탁드립니다.
```suggestion
    children: React.ReactNode;
```

Reply:
찾아본 결과 제안해주신 대로 children으로 받으면 더 좋을 것 같습니다. 수정했습니다.